<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lscpedit</class>
 <widget class="QMainWindow" name="lscpedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>lscpedit</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="2">
              <widget class="QPushButton" name="editMap_pushButton">
               <property name="text">
                <string>Edit MAP</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QPushButton" name="newMap_pushButton">
               <property name="text">
                <string>New MAP</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Map :</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QPushButton" name="deletMap_pushButton">
               <property name="text">
                <string>Delete MAP</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="3">
              <widget class="QComboBox" name="map_comboBox"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="1" column="0">
              <widget class="QPushButton" name="newItem_pushButton">
               <property name="text">
                <string>Add  </string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QPushButton" name="saveItem_pushButton">
               <property name="text">
                <string>Save</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="deleteItem_pushButton">
               <property name="text">
                <string>Delete </string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="clearAll_pushButton">
               <property name="text">
                <string>New</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Name :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="5">
            <widget class="QPushButton" name="selectInstFile_pushButton">
             <property name="text">
              <string>Select ..</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="4">
            <widget class="QLineEdit" name="instFilePath_lineEdit"/>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QComboBox" name="engine_comboBox"/>
           </item>
           <item row="1" column="4" colspan="2">
            <widget class="QComboBox" name="loadMode_comboBox"/>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Load mode :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" colspan="4">
            <widget class="QLineEdit" name="nameGig_lineEdit"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Filename :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Engine :</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QPushButton" name="resetName_pushButton">
             <property name="text">
              <string>Reset</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="4" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>LSB</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="msb_spinBox"/>
           </item>
           <item row="4" column="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>MIDI Program</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Volume</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Bank</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="lsb_spinBox"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>MSB</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QSpinBox" name="bank_spinBox"/>
           </item>
           <item row="4" column="3">
            <widget class="QSpinBox" name="prog_spinBox"/>
           </item>
           <item row="0" column="1">
            <widget class="QDoubleSpinBox" name="volume_doubleSpinBox"/>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="autoCalc_checkBox">
             <property name="text">
              <string>Auto Calc</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As ..</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
